bin/e2e-test-client/src/tests/test_data/arbitrary_tx.json

```diff
@@ -17,7 +17,7 @@
         0,
         0,
         0,
-        100000
+        1562622
```

Comment:
Unfortunately, these are fragile and will need to change any time we change the gas price algo.

Reply:
We should document these somehow, maybe a `README.md` in the `test_data` folder.

Reply:
Why we can't use 5M or 30M instead?

Reply:
We can. It is all arbitrary, so I just went with the least-arbitrary value. I guess we could avoid some fragility by doing a large value, but it won't be clear to engineers in the future why we used that value.

Reply:
I just bumped the value up to 30M. Idk if we need documentation for it any more than the other values. Happy to add something if you still think I should @Dentosal I get your point if we are doing a specific number, but now it's just a tx with plenty allocated.